notebooks/text_classification/solutions/text_generation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<ul><li> "&nbsp;Create training examples and targets" -&gt; "&nbsp;Create training examples and targets for text generation"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>